### DISCUSSION TITLE
Internal Server Error when using preview comments in custom domain alias

### DISCUSSION THREAD

**Discussion by AllanOliveiraM** [vercel/community#1562] [Help] [answered]
### Summary

# Internal Server Error when using preview comments in custom domain alias

I'm having trouble using comments in preview deployments.

I have comments enabled at both the team level and the project specific level.

The error occurs in the popup when I try to login to my vercel account from the widget.

### Additional information

I've tried things like:

- clear cache
- incognito tab
- other network
- other pc

## I tried to capture something to debug, but I didn't find much useful information:

```html
<script id="__NEXT_DATA__" type="application/json">{"props":{"pageProps":{"statusCode":500}},"page":"/_error","query":{},"buildId":"In8sLJjSTAzUlLJgYqjB0","nextExport":true,"isFallback":false,"gip":true,"scriptLoader":[]}</script>
```
Apparently sentry has been capturing this:

![image](https://user-images.githubusercontent.com/41436010/219791147-ae0f0d7e-a2e8-43d3-a264-1049e5a21d45.png)

The error:

![image](https://user-images.githubusercontent.com/41436010/219789248-11bcc533-9a22-4031-9637-4f01f5f6bd9d.png)

### Example

https://vercel.live/api/feedback/auth/initiate?id=KhiVt-bkplpdtWP&hostname=app-develop.granosafe.com.br

**Comment by amyegan** [comment]
Thanks for reporting this, Allan. I shared it with the team and we're looking into the cause

**Comment by amyegan** [comment, marked as answer]
A fix was deployed. Please try again and let me know if you continue seeing this issue

**Reply by AllanOliveiraM** [reply]
It worked :) Thank you very much
